minicli/__init__.py

```diff
@@ -31,31 +27,20 @@ def __call__(self, *args, **kwargs):
         except KeyboardInterrupt:
             pass
 
-    def invoke(self, parsed):
+    def invoke(self, parsed, **globals):
```

Comment:
Overriding the existing `globals` doesn't smell good to me. Proposition: `shared`?

Reply:
Sounds good, fixed :)